reference/conanfile.rst

```diff
@@ -762,7 +761,7 @@ SystemPackageTool methods:
 - **update()**: Updates the system package manager database.
```

Comment:
Probably clarify that update() is already automatically called before first "install"